src/EditorFeatures/CSharpTest/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -9012,64 +9460,111 @@ public static void M1()
 }
 
-class C : ITest
+class C : {|#0:ITest|}
 {
-    void ITest.M1()
+    void ITest.{|#1:M1|}()
     {
         throw new System.NotImplementedException();
     }
 }
-", parseOptions: CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.Preview), index: 1, title: FeaturesResources.Implement_all_members_explicitly);
+",
+                    },
+                    ExpectedDiagnostics =
+                    {
+                        // /0/Test0.cs(7,11): error CS0535: 'C' does not implement interface member 'ITest.M1()'
+                        DiagnosticResult.CompilerError("CS0535").WithLocation(0).WithArguments("C", "ITest.M1()"),
+                        // /0/Test0.cs(9,16): error CS0539: 'C.M1()' in explicit interface declaration is not found among members of the interface that can be implemented
+                        DiagnosticResult.CompilerError("CS0539").WithLocation(1).WithArguments("C.M1()"),
+                    },
+                },
+                CodeActionVerifier = (codeAction, verifier) => verifier.Equal(FeaturesResources.Implement_all_members_explicitly, codeAction.Title),
+                CodeActionEquivalenceKey = "True;False;False:global::ITest;TestProject;Microsoft.CodeAnalysis.ImplementInterface.AbstractImplementInterfaceService+ImplementInterfaceCodeAction;",
+                CodeActionIndex = 1,
+
+                // 🐛 This code fix is broken due to a missing 'static' keyword
+                CodeFixTestBehaviors = CodeFixTestBehaviors.FixOne,
+            }.RunAsync();
         }
 
         [WorkItem(53925, "https://github.com/dotnet/roslyn/issues/53925")]
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsImplementInterface)]
         public async Task TestStaticAbstractInterfaceMember_ImplementAbstractly()
         {
-            await TestInRegularAndScriptAsync(@"
+            await new VerifyCS.Test
+            {
+                ReferenceAssemblies = ReferenceAssemblies.Net.Net50,
+                LanguageVersion = LanguageVersion.Preview,
+                TestCode = @"
 interface ITest
 {
     static abstract void M1();
 }
 
-abstract class C : [|ITest|]
+abstract class C : {|CS0535:ITest|}
 {
 }
 ",
-@"
+                FixedState =
+                {
+                    Sources =
+                    {
+                        @"
 interface ITest
 {
     static abstract void M1();
 }
 
 abstract class C : ITest
 {
-    public abstract static void M1();
+    public abstract static void {|#0:M1|}();
 }
-", parseOptions: CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.Preview), index: 1, title: FeaturesResources.Implement_interface_abstractly);
+",
+                    },
+                    ExpectedDiagnostics =
+                    {
+                        // /0/Test0.cs(9,33): error CS0112: A static member cannot be marked as 'abstract'
+                        DiagnosticResult.CompilerError("CS0112").WithLocation(0).WithArguments("abstract"),
```

Comment:
🐛 Static members cannot be generated abstractly. Will file a follow-up issue.

Reply:
@sharwell  I believe I already fixed this in a PR that's not yet merged.

Reply:
➡️ https://github.com/dotnet/roslyn/issues/54328